.github/workflows/deploy_staging.yml

```diff
@@ -0,0 +1,63 @@
+name: Deploy Mapping Viz Staging
+
+on:
+    # Run this workflow on push to master (incl PR merges)
+    push:
+        branches: [ master ]
+
+    # Allow running this workflow manually from the Actions tab
+    workflow_dispatch:
+
+jobs:
+  build_staging:
+    name: Build Staging
+    runs-on: ubuntu-latest
+    env:
+      HEAD_COMMIT: ${{ github.sha }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: azure/login@v1
+      with:
+          creds: ${{ secrets.AZURE_STATIC_SITES }}
+
+    - name: Node.js build
+      uses: actions/setup-node@v1
+      with:
+        node-version: '16.x'
+    - run: yarn install --frozen-lockfile
+    - run: yarn build
+      env:
+        REACT_APP_ENV: development
```

Comment:
I'm not sure what this does but `development` looks like a strange setting for a staging deploy.

Reply:
The Alice Readme says this tells the Panoptes API client to use the staging API. That's confusing!

Might be worth adding this link to the Readme. 

https://github.com/zooniverse/panoptes-javascript-client/pull/128

Reply:
In this case, mapping-viz-tools is fetching from mapping-viz-functions instead of Panoptes API. It's okay for the staging site to fetch from mapping-viz-functions' prduction database so I suggest removing REACT_APP_ENV.